toolium/utils/dataset.py

```diff
@@ -565,14 +569,35 @@ def get_translation_by_poeditor_reference(reference, context):
     return translation
 
 
+def set_base64_path(path):
+    """
+    Set a relative path to be used as the base for the file path specified in the BASE64 mapping pattern.
+
+    :param path: relative path to be used as base for base64 mapping
+    """
+    global base_base64_path
+    base_base64_path = path
+
+
+def set_file_path(path):
+    """
+    Set a relative path to be used as the base for the file path specified in the FILE mapping pattern.
+
+    :param path: relative path to be used as base for file mapping
+    """
+    global base_file_path
+    base_file_path = path
+
+
 def get_file(file_path):
     """
-    Return the content of a file given its path.
+    Return the content of a file given its path. If a base path was previously set by using
+    the set_file_path() function, the file path specified must be relative to that path.
 
     :param file path: file path using slash as separator (e.g. "resources/files/doc.txt")
     :return: string with the file content
     """
-    file_path_parts = file_path.split("/")
+    file_path_parts = [os.getcwd()] + (base_file_path + file_path).split("/")
```

Comment:
instead of adding both lists, getcwd could be moved to next join
`file_path = os.path.join(os.getcwd(), *file_path_parts)`
or
`file_path = os.path.abspath(os.path.join(*file_path_parts))`

Reply:
Oook, done.